src/Form/DeleteMultiple.php

```diff
@@ -0,0 +1,204 @@
+          $names = [];
+          foreach ($languages as $translation_langcode => $language) {
+            $names[] = $language->getName();
+            unset($items[$id . ':' . $translation_langcode]);
+          }
+          $items[$default_key] = [
+            'label' => [
+              '#markup' => $this->t('@label (Original translation) - <em>The following content translations will be deleted:</em>', ['@label' => $log->label()]),
+            ],
+            'deleted_translations' => [
+              '#theme' => 'item_list',
+              '#items' => $names,
+            ],
+          ];
+        }
+        elseif (!isset($items[$default_key])) {
+          $items[$key] = $log->label();
+        }
+      }
+    }
+
+    $form['logs'] = array(
+      '#theme' => 'item_list',
+      '#items' => $items,
+    );
+    $form = parent::buildForm($form, $form_state);
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    if ($form_state->getValue('confirm') && !empty($this->logInfo)) {
+      $total_count = 0;
+      $delete_logs = [];
+      /** @var \Drupal\Core\Entity\ContentEntityInterface[][] $delete_translations */
+      $delete_translations = [];
+      /** @var \Drupal\log\LogInterface[] $logs */
+      $logs = $this->storage->loadMultiple(array_keys($this->logInfo));
+
+      foreach ($this->logInfo as $id => $langcodes) {
+        foreach ($langcodes as $langcode) {
+          $log = $logs[$id]->getTranslation($langcode);
+          if ($log->isDefaultTranslation()) {
+            $delete_logs[$id] = $log;
+            unset($delete_translations[$id]);
+            $total_count += count($log->getTranslationLanguages());
+          }
+          elseif (!isset($delete_logs[$id])) {
+            $delete_translations[$id][] = $log;
+          }
+        }
+      }
+
+      if ($delete_logs) {
+        $this->storage->delete($delete_logs);
+        $this->logger('log')->notice('Deleted @count posts.', array('@count' => count($delete_logs)));
+      }
+
+      if ($delete_translations) {
+        $count = 0;
+        foreach ($delete_translations as $id => $translations) {
+          $log = $logs[$id]->getUntranslated();
+          foreach ($translations as $translation) {
+            $log->removeTranslation($translation->language()->getId());
+          }
+          $log->save();
+          $count += count($translations);
+        }
+        if ($count) {
+          $total_count += $count;
+          $this->logger('log')->notice('Deleted @count content translations.', array('@count' => $count));
+        }
+      }
+
+      if ($total_count) {
+        drupal_set_message($this->formatPlural($total_count, 'Deleted 1 post.', 'Deleted @count posts.'));
+      }
+
+      $this->tempStoreFactory->get('log_multiple_delete_confirm')->delete(\Drupal::currentUser()->id());
+    }
+
+    $form_state->setRedirect('system.admin_content');
```

Comment:
The same here... or better - do something like here: https://github.com/zaporylie/log/blob/master/src/Form/LogDeleteForm.php#L56